src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -54,7 +55,47 @@ public sealed class CSharpCompilationOptions : CompilationOptions, IEquatable<CS
             StrongNameProvider strongNameProvider = null)
             : this(outputKind, false, moduleName, mainTypeName, scriptClassName, usings, optimizationLevel, checkOverflow, allowUnsafe,
                    cryptoKeyContainer, cryptoKeyFile, cryptoPublicKey, delaySign, platform, generalDiagnosticOption, warningLevel,
+                   specificDiagnosticOptions, concurrentBuild, deterministic,
+                   extendedCustomDebugInformation: true,
```

Comment:
Why do we have explicit values here VS just chaining to the ctor with all of the defaults and letting them win?  That strategy will put the default values in only a single location.